AFQ/api.py

```diff
@@ -233,8 +280,31 @@ def __init__(self,
 
         self.force_recompute = force_recompute
 
+        if target_b_val is not None:
+            self.target_b_upper = target_b_val + target_b_sensitivity
+            self.target_b_lower = target_b_val - target_b_sensitivity
+        else:
+            self.target_b_upper = None
+            self.target_b_lower = None
+
         self.wm_criterion = wm_criterion
-        self.use_prealign = use_prealign
+        self.reg_subject = reg_subject
+        self.reg_template = reg_template
+        self.mask_template = mask_template
+        if reg_subject == 'subject_sls' or reg_template == 'hcp_atlas':
+            if reg_template != 'hcp_atlas':
+                self.logger.error(
+                    "If reg_subject is 'subject_sls',"
+                    + " reg_template must be 'hcp_atlas'")
+            if reg_subject != 'b0':
```

Comment:
Shouldn't this be as follows?

```suggestion
            if reg_subject != 'subject_sls':
```

Reply:
good catch!